### ISSUE TITLE
Telemetry: refine duration of crc start

### ISSUE THREAD

**Issue by guillaumerose** [code-ready/crc#1907]
Currently the duration we send to segment includes the wait time of the pull secret.
Also, when the cluster is already started, the duration is also very short as we only display the end message. 

We should be able to select only the cluster creation duration.

**Comment by deboer-tim** [comment]
I agree we don't want to count time spent waiting for the user prompt as time against crc start, since that's variable and not in our control. Otherwise we should capture the full time though - from the user's perspective that is how long they're waiting.

Capturing _additional_ times like cluster creation would be great though, since that would give us a better breakdown of where the time is spent, how much of it is more directly in our control, or where we need to focus our efforts.

Going out on a limb here, but my last install had a long time where the cluster was up and crc start returned, but the console wasn't accessible for several minutes. Ideally we'd be able to capture this time as well. :)
